inscopix/isxcore
# IT-1225: Add initial prototype

## What type of PR is this? (check all applicable)


## Description
Add initial prototype of isxcore repo.

## Related Issues & Documents

- https://inscopix.atlassian.net/wiki/x/A4Bow

## Testing Instructions

I've tested on the following platforms:
* macOS 13
* Ubuntu 20.04
* Windows 11

1. Follow the setup instructions in the README
2. Ensure you have the third party and test data folders used to build/test IDPS available on your machine
3. Build the lib and run the unit tests: `make test THIRD_PARTY_DIR=/path/to/third/party/dir TEST_DATA_DIR=/path/to/test/data/dir`

## Added/updated tests?
* Moved all unit tests from IDPS to this repo and verified all tests pass on all supported platforms
